<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>statisticAnalysisClass</class>
 <widget class="QWidget" name="statisticAnalysisClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>statisticAnalysis</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <widget class="QLineEdit" name="lineEditVector">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>输入矢量：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditResult">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>输出分析结果：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openFile">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="login.qrc">
        <normaloff>:/login/res/openFile.png</normaloff>:/login/res/openFile.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="saveFile">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="login.qrc">
        <normaloff>:/login/res/saveFile.png</normaloff>:/login/res/saveFile.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="begin">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>116</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="login.qrc"/>
 </resources>
 <connections/>
</ui>
